osidb/models.py

```diff
@@ -696,6 +732,24 @@ def _validate_cvss3(self):
                 "CVSSv3 score is missing",
             )
 
+    def _validate_major_incident_state(self):
+        """
+        Checks that a flaw that is Major Incident has a valid state.
+        Valid states are MI_APPROVED or MI_CISA_APPROVED.
```

Comment:
I do not understand this validation. It seems to me that it checks whether **bzimport** works correctly and maps the Bugzilla flags to the boolean the right way. Or when should this alert trigger?

Reply:
I would alert on `MI_INVALID` only. Everything else is valid. The boolean should not play a role in it. It should be soon a property based on the MI state anyway so no need to check it.